doc/portability/portability.rst

```diff
@@ -0,0 +1,12 @@
+.. _portability:
+
+Portability Guide
```

Comment:
This title bothers me.  I think the document(s) in this section are about APIs from other RTOS systems that are provided by Zephyr to aid porting efforts from those RTOSes, right?  If so, a better title and description are warranted, maybe:

    .. _porting_to_zephyr:

    Porting Applications to Zephyr
    ##############################

    This section describes standard API interfaces provided by Zephyr
    specifically to ease porting applications originally written for other RTOS systems.

I'd also recommend not creating a new folder for these documents and just put them in the existing ``porting`` folder.

Reply:
sure. i agree. i'll modify and resubmit.